src/main/java/bio/terra/service/resourcemanagement/MetadataDataAccessUtils.java

```diff
@@ -24,11 +40,22 @@
   private static final String BIGQUERY_TABLE_ID = "<dataset_id>.<table>";
   private static final String BIGQUERY_BASE_QUERY = "SELECT * FROM `<table_address>` LIMIT 1000";
 
+  private static final String AZURE_BLOB_TEMPLATE = "parquet/<table>";
+  private static final String AZURE_DATASET_ID = "<storageAccount>.<dataset>";
+
   private static final String DEPLOYED_APPLICATION_RESOURCE_ID =
       "/subscriptions/<subscription>/resourceGroups"
           + "/<resource_group>/providers/Microsoft.Solutions/applications/<application_name>";
 
-  private MetadataDataAccessUtils() {}
+  private static ResourceService resourceService;
+
+  private static AzureBlobStorePdao azureBlobStorePdao;
```

Comment:
In general, you shouldn't have non-final `static` fields. In this case these are also injected objects that have state, so it doesn't make sense to use `static` fields for them.